CAP/gap/InstallAdds.gi

```diff
@@ -104,18 +104,6 @@ InstallGlobalFunction( CapInternalInstallAdd,
         argument_list := [ 1 .. Length( filter_list ) ];
     fi;
     
-    if record.return_type = "object" then
```

Comment:
The position of this `add_function` variable was on purpose, actually, as then the if is only carried out at install time, not at every call of the categorical operation.

Reply:
Ah, I see, should be fixed.

Reply:
Thanks :)